docs/configuration.md

```diff
@@ -196,6 +196,8 @@ As an example, the following would configure Ruff to:
 
     [tool.ruff.format]
     # 5. Use single quotes for non-triple-quoted strings.
+    #    (Note this only applies to the ruff formatter. To configure
+    #    the ruff linter to also use single quotes, see [tool.ruff.lint.flake8-quotes])
```

Comment:
I'd prefer not to introduce this complexity in the example... What if we change "5. Use single quotes for non-triple-quoted strings." to something like "5. Use single quotes in `ruff format`."?

Reply:
(Most users won't / shouldn't need the `flake8-quotes` rules at all, especially if they're using the formatter.)